cypress-tests/cypress/support/utils.ts

```diff
@@ -80,6 +80,48 @@ export const createPaymentBody = {
 
 }
 
+export const createPaymentBodyWithoutBillingAddress = {
```

Comment:
you can reuse the exisiting create body and remove billing and shipping from that object.